packages/nimble-components/src/wafer-map/tests/types.spec.ts

```diff
@@ -0,0 +1,15 @@
+import type { WaferMapOrientation, WaferMapQuadrant } from '../types';
+
+describe('WaferMap type', () => {
```

Comment:
reminder: If we decide to follow the recommendation for a `WaferMapColorsScaleMode` we may need a new enum test

Reply:
I added